test/server.test.js

```diff
@@ -0,0 +1,30 @@
+// import puppeteer from 'puppeteer';
+
+// const pageURL = 'http://localhost:3001';
+
+// let page;
+// let browser;
+// const width = 1280;
+// const height = 720;
+
+// beforeAll(async () => {
+//   browser = await puppeteer.launch({
+//     headless: false,
+//     args: [`--window-size=${width},${height}`],
+//   });
+//   page = await browser.newPage();
+//   await page.setViewport({
+//     width,
+//     height
+//   });
+// });
+
+// afterAll(() => {
+//   browser.close();
+// });
+
+describe('this is a test', () => {
+  test('a dumb test', () => {
+    
+  });
+});
```

Comment:
Any reason for this file at the moment too? Is it just a place holder?

Reply:
I don't have any end-to-end tests yet, this is just a test to allow the file to pass. Will be filled in after I finish the calendar